scripts/release_notes_generator.py

```diff
@@ -0,0 +1,128 @@
+    'bug': 'Bugs Fixed',
+    'internal': 'Internal',
+    'maintenance': 'Maintenance',
+    'customer success': 'Customer Success',
+    'documentation': 'Documentation',
+    'misc': 'Miscellaneous',
+}
+ISSUE_LABELS = [
+    'documentation',
+    'maintenance',
+    'internal',
+    'bug',
+    'feature request',
+    'customer success',
+]
+NEW_LINE = '\n'
+GITHUB_URL = 'https://api.github.com/repos/sdv-dev/sdgym'
+GITHUB_TOKEN = os.getenv('GH_ACCESS_TOKEN')
+
+
+def _get_milestone_number(milestone_title):
+    url = f'{GITHUB_URL}/milestones'
+    headers = {'Authorization': f'Bearer {GITHUB_TOKEN}'}
+    query_params = {'milestone': milestone_title, 'state': 'all', 'per_page': 100}
+    response = requests.get(url, headers=headers, params=query_params)
+    body = response.json()
+    if response.status_code != 200:
+        raise Exception(str(body))
+    milestones = body
+    for milestone in milestones:
+        if milestone.get('title') == milestone_title:
+            return milestone.get('number')
+    raise ValueError(f'Milestone {milestone_title} not found in past 100 milestones.')
+
+
+def _get_issues_by_milestone(milestone):
+    headers = {'Authorization': f'Bearer {GITHUB_TOKEN}'}
+    # get milestone number
+    milestone_number = _get_milestone_number(milestone)
+    url = f'{GITHUB_URL}/issues'
+    page = 1
+    query_params = {'milestone': milestone_number, 'state': 'all'}
+    issues = []
+    while True:
+        query_params['page'] = page
+        response = requests.get(url, headers=headers, params=query_params)
+        body = response.json()
+        if response.status_code != 200:
+            raise Exception(str(body))
+        issues_on_page = body
+        if not issues_on_page:
+            break
+        issues.extend(issues_on_page)
+        page += 1
+    return issues
+
+
+def _get_issues_by_category(release_issues):
+    category_to_issues = defaultdict(list)
+    for issue in release_issues:
+        issue_title = issue['title']
+        issue_number = issue['number']
+        issue_url = issue['html_url']
+        line = f'* {issue_title} - Issue [#{issue_number}]({issue_url})'
+        assignee = issue.get('assignee')
+        if assignee:
+            login = assignee['login']
+            line += f' by @{login}'
+        # Check if any known label is marked on the issue
+        labels = [label['name'] for label in issue['labels']]
+        found_category = False
+        for category in ISSUE_LABELS:
+            if category in labels:
+                category_to_issues[category].append(line)
+                found_category = True
+                break
+        if not found_category:
+            category_to_issues['misc'].append(line)
+    return category_to_issues
+
+
+def _create_release_notes(issues_by_category, version, date):
+    title = f'## v{version} - {date}'
+    release_notes = f'{title}{NEW_LINE}{NEW_LINE}'
+    for category in ISSUE_LABELS + ['misc']:
```

Comment:
SDV-Enterprise uses [another list ](https://github.com/datacebo/SDV-Enterprise/blame/e4800d083a539232b7456141d35b173f2ba8601e/scripts/release_notes_generator.py#L26) here.

Reply:
@R-Palazzo do these changes need to be propagated to any other libraries?

Reply:
Good catch, thanks @fealho.
Yes, if we want to use what's in Enterprise, SDMetrics and SDGym must be updated (other libraries already have the changes)
I have a PR on both libraries that are related so I will include the changes @amontanez24